### ISSUE TITLE
[FEATURE] Improve A11y and enable Reader view

### ISSUE THREAD

**Issue by saenyakorn** [thinc-org/cugetreg#471]
**Is your feature request related to a problem? Please describe.**

Most modern browser has the feature called `Reader View` for visually impaired. For example, Medium or other web blog. To toggle Reader Viewer, you need to click paper-like icon near the browser search bar.

<img width="1680" alt="Screenshot 2566-01-29 at 23 48 46" src="https://user-images.githubusercontent.com/33742791/215341273-b5442255-3483-4364-89cc-003d6f83271a.png">

So, the Course Detail page, with many text and detail in its, should has more readability and should be improved accessibility for SEO purpose. 

**Describe the solution you'd like**

Improve a11y and then enable reader view for Course Detail pages

**Describe alternatives you've considered**

**Additional context**

**Comment by bombnp** [comment]
So we're doing this for SEO purposes lmao
